changes/changes-8.8.0.md

```diff
@@ -41,6 +41,7 @@ Release Notes.
 * Optimize okhttp 3.x 4.x plugin to get span time cost precisely
 * Adapt message header properties of RocketMQ 4.9.x
 * Fix httpasyncclient-4.x-plugin's memory leak risk
+* Fix the bug that httpasyncclient-4.x-plugin  use same request produce different trace
```

Comment:
This is the 8.8.0 change log. Your change log is in the root folder.

And this is recommended change log and PR title.
```suggestion
* Fix the bug that httpasyncclient-4.x-plugin  puts the dirty tracing context in the connection context.
```